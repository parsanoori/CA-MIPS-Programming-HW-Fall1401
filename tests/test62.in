77
273181522
461949612
225354249
60262022
829024720
537215332
352468473
747285618
332841295
704714727
941895905
571286168
702154540
423381205
253993223
158251273
451684741
611892278
13585429
832669631
902037616
707745091
465899259
166356059
359083053
442965427
1008887398
195830079
587993169
870015311
41253018
300888334
788167350
993851757
504439357
1028203274
167139224
357728344
1024363374
823178529
523871015
143286174
1033758766
933587027
839471681
188231482
541470776
648150934
830297941
379719047
634389030
417377545
884782647
924041393
301616754
321073443
599959582
869536977
811289200
394556071
110353209
649592978
725029931
774609371
774091918
259980391
451747960
640347484
339421519
681433535
317836058
598330540
643424684
129482859
718664025
374008132
582291347