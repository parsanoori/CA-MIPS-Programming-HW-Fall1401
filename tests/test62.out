13585429
41253018
60262022
110353209
129482859
143286174
158251273
166356059
167139224
188231482
195830079
225354249
253993223
259980391
273181522
300888334
301616754
317836058
321073443
332841295
339421519
352468473
357728344
359083053
374008132
379719047
394556071
417377545
423381205
442965427
451684741
451747960
461949612
465899259
504439357
523871015
537215332
541470776
571286168
582291347
587993169
598330540
599959582
611892278
634389030
640347484
643424684
648150934
649592978
681433535
702154540
704714727
707745091
718664025
725029931
747285618
774091918
774609371
788167350
811289200
823178529
829024720
830297941
832669631
839471681
869536977
870015311
884782647
902037616
924041393
933587027
941895905
993851757
1008887398
1024363374
1028203274
1033758766